### ISSUE TITLE
The schema for Microsoft.Insights/metricAlerts does not include the definition for creating an alert for a web availability test

### ISSUE THREAD

**Issue by MWL88** [Azure/azure-resource-manager-schemas#1074]
When editing an ARM template in VS Code to create an alert for a web availability test, VS Code reports a warning for the `criteria` property:
```
Value must conform to exactly one of the associated schemas
|   Value must be one of the following values: "Microsoft.Azure.Monitor.SingleResourceMultipleMetricCriteria" at #/resources/6/properties/criteria/odata.type
|   or
|   Value must be one of the following values: "Microsoft.Azure.Monitor.MultipleResourceMultipleMetricCriteria" at #/resources/6/properties/criteria/odata.type
arm-template (schema)
```

The `criteria` property for a web availability test alert is type `Microsoft.Azure.Monitor.WebtestLocationAvailabilityCriteria`, which is neither of the types specified in the warning.

The required type for a web availability test alert is specified in the [REST API definition](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json#L657) but it is not included in the [ARM schema definition](https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2018-03-01/Microsoft.Insights.json#L566).

**Comment by MWL88** [comment]
Hi @apclouds, please will you progress / triage this issue.
Thanks

**Comment by anthony-c-martin** [comment]
Microsoft.Insights are not onboarded for automatic generation; we'll follow up and assign an owner to fix this.

**Comment by anthony-c-martin** [comment]
FYI, this should be fixed with #1141

**Comment by MWL88** [comment]
Thanks
